<ui version="4.0" >
 <comment>Widget used for creating an account manually. </comment>
 <class>GuiAddAccountBase</class>
 <widget class="QWidget" name="GuiAddAccountBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>GuiAddAccountBase</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="textLabel1" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>&lt;h2>Add an account:&lt;/h2></string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer2" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>241</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="spacer3" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>21</width>
       <height>31</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer name="spacer8" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>71</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="textLabel2" >
       <property name="text" >
        <string>Attention: In most cases you will not need to add accounts manually, because the account list is received from the bank, when you update your data. However, the list may not have been sent by the bank. In this unusual case you may add an account by hand.</string>
       </property>
       <property name="wordWrap" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer7" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>61</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer name="spacer5" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>81</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Add Account for Bank:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelBank" >
         <property name="text" >
          <string>Bankdata</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel3_2_2" >
         <property name="text" >
          <string>Accountname:</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
         <property name="buddy" >
          <cstring>lineEdit1_2_2</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditAccountName" />
       </item>
       <item>
        <widget class="QLabel" name="textLabel3" >
         <property name="text" >
          <string>Account Id:</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
         <property name="buddy" >
          <cstring>lineEdit1</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditAccountId" />
       </item>
       <item>
        <widget class="QLabel" name="textLabel3_3" >
         <property name="text" >
          <string>Account Sub Id:</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
         <property name="buddy" >
          <cstring>lineEdit1_3</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditAccountSubId" />
       </item>
       <item>
        <widget class="QLabel" name="textLabel3_2" >
         <property name="text" >
          <string>Owner:</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
         <property name="buddy" >
          <cstring>lineEdit1_2</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditName" />
       </item>
      </layout>
     </item>
     <item>
      <spacer name="spacer5_2" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>81</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="spacer4" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>21</width>
       <height>31</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="buttonBack" >
       <property name="text" >
        <string>&amp;Back</string>
       </property>
       <property name="shortcut" >
        <string>Alt+B</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer1" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>201</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk" >
       <property name="text" >
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
